src/chains/ethereum/ethereum/tests/blockchain.test.ts

```diff
@@ -112,7 +112,12 @@ describe("blockchain", async () => {
       // assert that second block's timestamp is at least `blockTime` greater
       // than the first block's. meaning, these blocks weren't mined one after
       // the other
-      assert(timestamps[1] >= timestamps[0] + blockTime);
+      assert(
+        timestamps[1] >= timestamps[0] + blockTime,
+        `Unexpected timestamp - expected >= ${timestamps[0] + blockTime}, got ${
```

Comment:
I updated this message as I was debugging the change, and the error message wasn't helpful